### ISSUE TITLE
Missing measures

### ISSUE THREAD

**Issue by visr** [JuliaGeo/Shapefile.jl#28]
For measures, floats smaller than -1e38 are considered "no data".

Perhaps we should convert these to `missing`, and write `missing` to small floats.
This also goes for the `Mmin` and `Mmax` in the header, which will be missing if all measures are missing. If this is the right interpretation of the spec that is.

Came up in #25

**Comment by rafaqz** [comment]
Another option is to use a sentinel approach with a wrapper array type, and replace the missing values on the fly in `getindex`. This may have less performance cost for the cases where you don't actually use the measures data.

**Comment by visr** [comment]
Yeah that could work as well. I'm not aware of anyone having needed support for this yet, I don't know how common it is. Perhaps we can even get away with just documenting it, and/or reading as NaN.
